packages/elements-dev-portal/package.json

```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
-    "@stoplight/elements-core": "~7.1.6",
+    "@stoplight/elements-core": "^7.1.7",
```

Comment:
Nope :P

Reply:
Fuck lerna. I'm really looking for a nice monorepo tool 😅 Or maybe just moving back to linked versioning 😄